lib/pages/saved_places/saved_places.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:latlong/latlong.dart';
+
+import 'package:trufi_core/blocs/search_locations/search_locations_cubit.dart';
+import 'package:trufi_core/l10n/trufi_localization.dart';
+import 'package:trufi_core/pages/saved_places/location_tiler.dart';
+import 'package:trufi_core/widgets/dialog_edit_text.dart';
+
+import '../../trufi_models.dart';
+import '../../widgets/trufi_drawer.dart';
+import '../choose_location.dart';
+
+class SavedPlacesPage extends StatelessWidget {
+  static const String route = '/places';
+
+  const SavedPlacesPage({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final searchLocationsCubit = context.read<SearchLocationsCubit>();
+    final localization = TrufiLocalization.of(context);
+    final theme = Theme.of(context);
+    return Scaffold(
+      appBar: AppBar(title: Text(localization.menuYourPlaces)),
+      body: DefaultTabController(
+        length: 2,
+        child: Column(
+          children: [
+            const SizedBox(height: 10),
+            TabBar(
+              tabs: [
+                Container(
+                  padding: const EdgeInsets.all(10),
+                  child: Text(
+                    // TODO translate
```

Comment:
I would recommend directly use the localization else we will have a lot of work to find all the locations and understand how we can update them.

Reply:
I talked with @SamuelRioTz  and  We think that these translations can change and even some will no longer be used, to avoid the double work of modifying or deleting translations we plan to do them almost at the end.

Reply:
Ok, if you decided as a team I am fine with it.

Reply:
@md-weber  the translation is also part of the UI/UX changes probably the translations will be changed, so we talked for now is ok skip some translations, especially the new ones